packages/cli/src/lib/exec.js

```diff
@@ -4,14 +4,15 @@ import {
   getWebSideDefaultBabelConfig,
   registerApiSideBabelHook,
 } from '@redwoodjs/babel-config'
+import { resolveSourcePath } from '@redwoodjs/internal/dist/files'
 import { getPaths } from '@redwoodjs/project-config'
 
 export async function runScriptFunction({
   path: scriptPath,
   functionName,
   args,
 }) {
-  const script = require(scriptPath)
+  const script = require(resolveSourcePath(scriptPath))
```

Comment:
Does this work if the user did `yarn rw exec myScript.ts`?

Reply:
It does not -- will adjust.

Reply:
Fixed this to allow `yarn rw exec myScript` and `yarn rw exec myScript.ts` to succeed, and adjusted the detail in error messages appropriately for cases we can't resolve the path.